keps/sig-api-machinery/4008-crd-ratcheting/kep.yaml

```diff
@@ -35,6 +35,7 @@ latest-milestone: "v1.30"
 milestone:
   alpha: "v1.28"
   beta: "v1.30"
+  ga: "v1.32"
```

Comment:
This is a future release, is it worth adding now or wait until that cycle?

Reply:
would definitely wait.